### DISCUSSION TITLE
new syntax : #[weak]

### DISCUSSION THREAD

**Discussion by philippe06150** [gtk-rs/gtk4-rs#1774] [Q&A] [answered]
Hi,

going through clipboard example, I got an error with downloaded file from examples

```
   Compiling gtk_demo v0.1.0 (E:\RUST\gtk_demo)
error: Unexpected token `[weak]`
  --> src\bin\clipboard.rs:58:10
   |
58 |         #[weak]
   |          ^^^^^^
```

my toml file is

```
[dependencies]
gtk =    { git = "https://github.com/gtk-rs/gtk4-rs.git", package = "gtk4" , features = ["v4_14"]}
```

I update rust to 1.79, I noticed that #[weak], is a new syntax, so I suspect an issue concern, but i do not find the point ???

**Comment by sdroege** [comment, marked as answer]
You're probably missing a `cargo update -p glib`.

**Comment by philippe06150** [comment]
yes ... probably ;-)

it works now
